tasks/install/compiled_nginx.yml

```
@@ -35,8 +35,8 @@
   with_items: "{{nginx_sources}}"
 
 - name: Configure nginx source code
-  command: >
-    ./configure {{nginx_compile_flags}}
+  shell: >
+    {{nginx_configure_environ}} ./configure {{nginx_compile_flags}}
```

There is no need to fallback to `shell`. You could define the execution environment as:

```suggestion
  command: >
    ./configure {{nginx_compile_flags}}
  environment: "{{nginx_configure_environ}}"
```

Please note thay `nginx_configure_environ` should be a dict instead of a plain string

---

Are you sure? I checked the [documentation](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/command_module.html) and there was no environment arg

---

Yes, it's not part of the module, is a general [ansible feature](https://docs.ansible.com/ansible/latest/user_guide/playbooks_environment.html)